server/lib/utils.js

```diff
@@ -15,9 +15,22 @@ module.exports = {
         const svgFilePath = `/tmp/${md5}.svg`;
         const outputDir = `/tmp`;
         const outputFile = `${outputDir}/${md5}.png`;
-        fs.writeFileSync(svgFilePath, svg);
-        
-        return svg_to_png.convert(svgFilePath, outputDir)
-                .then(() => readFile(outputFile));
+
+        try {
+            // If file exists, return it
+            // Note: because we generate a md5 fingerprint based on the content of the svg,
+            //       any change in the svg (margin, size, number of backers, etc.) will force
+            //       the creation of a new png :-)
```

Comment:
Is there a mechanism to get rid of old files accumulating on the server? (don't have full picture of how you implemented this)